src/main/app/case/C100CaseApi.ts

```diff
@@ -67,11 +67,13 @@ export class CaseApi {
     caseId: string,
     caseData: Partial<CaseWithId>,
     returnUrl: string,
-    caseEvent: C100_CASE_EVENT
+    caseEvent: C100_CASE_EVENT,
+    additionalSettings: Record<string, any> | undefined
```

Comment:
what is this?

Reply:
why it is required?

Reply:
In order to store pcqId for applicant as part of case submission

Reply:
But why to store in a new prop in session?